blog/models.py

```diff
@@ -39,6 +39,7 @@ class Article(db.Document):
     author = db.ReferenceField(User)
     language = db.StringField(min_length=2, max_length=2, default='en')
     images = db.ListField()
+    is_published = db.BooleanField()
```

Comment:
@Ydrasil I would say it's the first day of publication.
We have the `modification_date` for other purposes.